### PR TITLE
add dns protocol config doc

### LINKED ISSUE #269: Add in doc about the DNS protocol
It would be nice to add the configuration options for the DNS protocols under the Configuration->Rrotocols section.

### PR DESCRIPTION
**Pull request by urso**
Add dns protocol analyser configuration documentation to
configuration.asciidoc

fixes #269

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -235,6 +239,36 @@ want to index the whole request. Note that for HTTP, the body is not included
...
 
 
+==== DNS configuration
+
+DNS protocol supports processing DNS message via TCP and UDP. Here is a sample configuration section for DNS:
+
+[source,yaml]
+------------------------------------------------------------------------------
+protocols:
+  http:
```
Comment (tsg) [inline docs/configuration.asciidoc]: http? :)

Comment (urso) [inline docs/configuration.asciidoc]: argh...

Comment (urso) [inline docs/configuration.asciidoc]: fixed

**Comment by tsg** [comment]
Needs merging :(

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -235,6 +239,36 @@ want to index the whole request. Note that for HTTP, the body is not included
 by default, only the HTTP headers.
 
 
+==== DNS configuration
+
+DNS protocol supports processing DNS message via TCP and UDP. Here is a sample configuration section for DNS:
```
Comment (andrewkroh) [inline docs/configuration.asciidoc]: Unfortunately, it only supports UDP at the moment.

Comment (urso) [inline docs/configuration.asciidoc]: Oh, ok. Thanks.

**Comment by andrewkroh** [comment]
LGTM. Thanks for adding this.

**Pull request merged**